nullaway/src/main/java/com/uber/nullaway/ErrorMessage.java

```diff
@@ -24,12 +24,21 @@
 
 /** Contains error message string to be displayed and the message type from {@link MessageTypes}. */
 public class ErrorMessage {
+
+  static final String COVERED = "(Covered) ";
```

Comment:
I definitely think we shouldn't alter the NullAway error messages when not running the autofixer. In fact, to make things easier to compare even when we are running the autofixer, I'd say we shouldn't change the text of the error messages either way (i.e. we shouldn't add "(Covered) "). Is this necessary for the autofixer to know which errors have an autofix and which do not? Or is this information already passed in a different way?

Reply:
Sure, this information is not passed to ```AutoFixer``` and this is just for our information. I will safely remove this.